inc/Config/QueryRunner/QueryRunner.php

```diff
@@ -166,6 +169,7 @@ protected function get_raw_response_data( HttpQueryInterface $query, array $inpu
 	protected function get_response_metadata( HttpQueryInterface $query, array $response_metadata, array $query_results ): array {
 		$age = intval( $response_metadata['age'] ?? 0 );
 		$time = time() - $age;
+		$count = count( $query_results );
```

Comment:
This was a minor change I had made. It's impact isn't that much, but every bit counts.

Reply:
I'm not sure how this has a performance impact, but it won't make it worse.